app/controllers/course/assessment/categories_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Course::Assessment::CategoriesController < Course::Assessment::Controller
```

Comment:
`index` and `update` are handled in assessment settings for both categories and tabs, but `create` and `destroy` are handled here (see routes). Seemed like better conceptual separation.